src/test/java/org/benetech/servicenet/mother/OrganizationMother.java

```diff
@@ -134,16 +136,21 @@ public static Organization createDefaultWithAllRelationsAndPersist(EntityManager
         Organization org = createDefault(DEFAULT_ACTIVE);
 
         org.setAccount(SystemAccountMother.createDefaultAndPersist(em));
-        org.setUserProfiles(Collections.singleton(UserMother.createDefaultAndPersist(em)));
```

Comment:
The entity manager was unable to modify sets created with Collections.singleton (it's immutable implementation of a set)